src/server.ts

```diff
@@ -1,5 +1,5 @@
 import * as dotenv from "dotenv";
-dotenv.config({ path: __dirname+'/.env' });
+dotenv.config();
```

Comment:
No Nico , the env is failing if we don't specify the path. Had this issue on render too and facing it locally too. 
```
dotenv.config({ path: __dirname+'/.env' });
```
using dotenv like above make sure it works.

Reply:
That is weird :S are you using Linux? In my Windows setup it fails when we specify the path :(

Reply:
I just pushed a new change reverting this, could you try to run this in your local? :)

Reply:
Yes I am running Linux . I can put it in try catch block i guess.

Reply:
I will read more about it and try to fix it in https://github.com/State-Channel-4/backend/tree/feat/nico/sync-contract